src/components/Header/Header.css

```diff
@@ -0,0 +1,20 @@
+.App {
+    text-align: center;
+    top: 5px;
+  }
+    
+.App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 80px;
+  }
+  
+  .App-header {
+    background-color: #222;
+    height: 80px;
+    padding: 6px;
+    color: white;
+  }
+  
+  .App-intro {
+    font-size: large;
+  }
```

Comment:
same shit kaip auksciau, plius indikuoja `no newline at end of file` - reiktu pasidomet `.editorconfig` arba `.gitattributes` failiukais

Reply:
https://help.github.com/articles/dealing-with-line-endings/